pkg/apmintegration/file.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ava-labs/apm/types"
+	"github.com/ava-labs/avalanche-cli/pkg/application"
+	"gopkg.in/yaml.v3"
+)
+
+func GetRepos(app *application.Avalanche) ([]string, error) {
```

Comment:
Note to myself but the majority of this file looks like good functionality that I should just end up exposing/refactoring this into the APM instead of having you to have to implement this with filesystem lookups upstream. Feels like avalanche-cli isn't the right place to have this code, I'll probably end up pulling this into apm at some point.

Reply:
Yep, I'd agree with that. We'll leave them here for now then remove when you update the apm.

Reply:
https://github.com/ava-labs/apm/issues/24